charts/datarepo/Chart.lock

```diff
@@ -17,5 +17,8 @@ dependencies:
 - name: de-elasticsearch
   repository: https://broadinstitute.github.io/datarepo-helm/
   version: 0.1.4
+- name: postgres
```

Comment:
I don't know if updating chart lock manually like this will work.  Isn't the digest wrong then?

Reply:
I actually don't know if that'll break things but regenerated the lockfile just in case, probably a good idea anyway